### ISSUE TITLE
Default import from `@storybook/addons` breaks in `storybook@7.x`

### ISSUE THREAD

**Issue by joshacheson** [storybookjs/testing-react#118]
**Describe the bug**
Per the [storybook 7 migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#no-more-default-export-from-storybookaddons) the default export from `@storybook/addons` is no longer available. We should move to the named import. 

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#no-more-default-export-from-storybookaddons

**To Reproduce**
Install storbook @7.x and attempt to use @storybook/testing-react.

**Comment by IanVS** [comment]
I'm also seeing this error with alpha.46:

```
'defaultDecorateStory' is not exported by node_modules/@storybook/client-api/dist/index.mjs, imported by node_modules/@storybook/testing-react/dist/testing-react.esm.js
file: /Users/ianvs/code/defined/webclient/node_modules/@storybook/testing-react/dist/testing-react.esm.js:1:28
1: import { combineParameters, defaultDecorateStory } from '@storybook/client-api';
                               ^
2: import addons, { mockChannel, HooksContext, applyHooks } from '@storybook/addons';
```

**Comment by IanVS** [comment]
This is fixed in version [v2.0.0-next.0](https://github.com/storybookjs/testing-react/releases/tag/v2.0.0-next.0).  Please let us know if you still have any problems after updating.
